quickstarts/microsoft.web/webapp-linux-managed-mysql/main.bicep

```diff
@@ -0,0 +1,149 @@
+@description('main.bicep')
+param siteName string = '${uniqueString(resourceGroup().name)}'
+
+@description('Database administrator login name')
+@minLength(1)
+param administratorLogin string
+
+@description('Database administrator password')
+@minLength(8)
+@secure()
+param administratorLoginPassword string
+
+@description('Azure database for mySQL compute capacity in vCores (2,4,8,16,32)')
+@allowed([
+  2
+  4
+  8
+  16
+  32
+])
+param dbSkucapacity int = 2
+
+@description('Azure database for mySQL sku name ')
+@allowed([
+  'GP_Gen5_2'
+  'GP_Gen5_4'
+  'GP_Gen5_8'
+  'GP_Gen5_16'
+  'GP_Gen5_32'
+  'MO_Gen5_2'
+  'MO_Gen5_4'
+  'MO_Gen5_8'
+  'MO_Gen5_16'
+  'MO_Gen5_32'
+])
+param dbSkuName string = 'GP_Gen5_2'
+
+@description('Azure database for mySQL Sku Size ')
+@allowed([
+  102400
+  51200
+])
+param dbSkuSizeMB int = 51200
+
+@description('Azure database for mySQL pricing tier')
+@allowed([
+  'GeneralPurpose'
+  'MemoryOptimized'
+])
+param dbSkuTier string = 'GeneralPurpose'
+
+@description('MySQL version')
+@allowed([
+  '5.6'
+  '5.7'
+])
+param mysqlVersion string = '5.7'
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Azure database for mySQL sku family')
+param databaseskuFamily string = 'Gen5'
+
+var databaseName = '${uniqueString(resourceGroup().id)}'
+var mysqlserverName = '${uniqueString(resourceGroup().id)}'
+var hostingPlanName = '${uniqueString(resourceGroup().id)}'
+
+resource hostingPlan 'Microsoft.Web/serverfarms@2020-06-01' = {
+  name: hostingPlanName
+  location: location
+  sku: {
+    tier: 'Standard'
+    name: 'S1'
+  }
+  kind: 'linux'
+  properties: {
+    reserved: true
+  }
+}
+
+resource site 'Microsoft.Web/sites@2021-02-01' = {
+  name: siteName
+  location: location
+  properties: {
+    siteConfig: {
+      linuxFxVersion: 'php|7.0'
+      ftpsState: 'FtpsOnly'
+      minTlsVersion: '1.2'
+      scmMinTlsVersion: '1.2'
+      connectionStrings: [
+        {
+          name: 'defaultConnection'
+          connectionString: 'Database=${databaseName};Data Source=${mysqlserver.properties.fullyQualifiedDomainName};User Id=${administratorLogin}@${mysqlserverName};Password=${administratorLoginPassword}'
+          type: 'MySql'
+        }
+      ]
+    }
+    serverFarmId: hostingPlan.id
+    httpsOnly: true
+  }     
+   identity: {
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

Reply:
I think that this section was the only one with the incorrect order. Should be resolved now.